adjust the solana ntt deployment docs to the new cli commands for setting the mint authority + demo-ntt-connect comment from vite to nextjs (#510)
wormhole-foundation/wormhole-docs: products/native-token-transfers/guides/deploy-to-solana.md

Before updating the mint authority, you must create metadata for your SPL token. You can visit this repository to see an example of [how to create metadata for your SPL token](https://github.com/wormhole-foundation/demo-metaplex-metadata/blob/main/src/token-metadata.ts){target=\_blank}.

Options to set the mint authority for your SPL token:

**For undeployed programs:**

- **Set to token authority PDA:**
```bash
ntt solana set-mint-authority --token INSERT_TOKEN_ADDRESS --manager INSERT_NTT_PROGRAM_ADDRESS --payer INSERT_KEYPAIR_JSON
```

- **Set to SPL Multisig:**

> 1. Create valid SPL Multisig:

Reviewer: ```suggestion
    1. Create valid SPL Multisig:
```

> ```bash

Reviewer: ```suggestion
    ```bash
```

> ntt solana create-spl-multisig INSERT_MINTER_PUBKEY_1 INSERT_MINTER_PUBKEY_2 ... --token INSERT_TOKEN_ADDRESS --manager INSERT_NTT_PROGRAM_ADDRESS --payer INSERT_KEYPAIR_JSON

Reviewer: ```suggestion
    ntt solana create-spl-multisig INSERT_MINTER_PUBKEY_1 INSERT_MINTER_PUBKEY_2 ... --token INSERT_TOKEN_ADDRESS --manager INSERT_NTT_PROGRAM_ADDRESS --payer INSERT_KEYPAIR_JSON
```

> ```

Reviewer: ```suggestion
    ```
```

2. Set to created SPL Multisig:

> ```bash

Reviewer: ```suggestion
    ```bash
```

> ntt solana set-mint-authority --token INSERT_TOKEN_ADDRESS --manager INSERT_NTT_PROGRAM_ADDRESS --multisig INSERT_MULTISIG_ADDRESS --payer INSERT_KEYPAIR_JSON

Reviewer: ```suggestion
    ntt solana set-mint-authority --token INSERT_TOKEN_ADDRESS --manager INSERT_NTT_PROGRAM_ADDRESS --multisig INSERT_MULTISIG_ADDRESS --payer INSERT_KEYPAIR_JSON
```

> ```

Reviewer: ```suggestion
    ```
```

**For deployed programs:**

- **Set to token authority PDA:**

```bash
ntt solana set-mint-authority --payer INSERT_YOUR_KEYPAIR_JSON
```

> 

Author: I left it out to try to limit the docs file as it is getting pretty big already. If we see the 'set to spl multisig' example above not being clear enough, we can add it afterwards. What do you think?

> 

Reviewer 2: Yeah, I agree.

!!! note

>     Checkout [this utility script](https://github.com/wormhole-foundation/demo-ntt-token-mint-authority-transfer/tree/main) for transferring token mint authority out of NTT.

Reviewer: ```suggestion
    Check out [this utility script](https://github.com/wormhole-foundation/demo-ntt-token-mint-authority-transfer/tree/main){target=\_blank} for transferring token mint authority out of NTT.
```
